metastore/scripts/upgrade/hive/hive-schema-3.0.0.hive.sql

```diff
@@ -0,0 +1,1334 @@
+-- HIVE system db
+
+CREATE DATABASE SYS;
```

Comment:
Let me see if I understand the question:

I think it does. It creates tables in sys, which are basically "views" on the underlying HMS db, meaning they don't have data, but a pointer to the HMS tables). This way admins can access them directly  from hive and users with permissions can look up info that's missing from the information schema (stats come to mind).

There's also an information_schema that is basically a set of pure hive views on top of that. 

The jdbc storage handler requires tables to be defined in hive for any RDBMS access. We could create views in HMS db and expose those through another set of tables alongside the ones in sys, but I find this cleaner (less dependency on each supported DB) and easier to maintain.